src/Typedefs/Test/TermParseWriteTests.idr

```diff
@@ -13,40 +13,40 @@ import Specdris.Spec
 
 %access public export
 
-roundtrip1 : (td : TDefR 0) -> Ty [] td -> Maybe ((Ty [] td), Bytes)
-roundtrip1 td x = deserialiseBinaryClosed td $ serializeBinaryClosed td x
-
-roundtrip2 : (td : TDefR 0) -> Bytes -> Maybe Bytes
-roundtrip2 td b = map (serializeBinaryClosed td . fst) (deserialiseBinaryClosed td b)
+-- roundtrip1 : (td : TDefR 0) -> Ty [] td -> Maybe ((Ty [] td), Bytes)
+-- roundtrip1 td x = deserialiseBinaryClosed td $ serialise [] [] td x
+--
+-- roundtrip2 : (td : TDefR 0) -> Bytes -> Maybe Bytes
+-- roundtrip2 td b = map (serialise [] [] td . fst) (deserialiseBinaryClosed td b)
```

Comment:
What's wrong with these?

Reply:
Nothing, but they weren't used. I should probably delete them and write some tests for them

Reply:
I wrote an issue about this #228